Vagrantfile

```diff
@@ -48,7 +50,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |c|
     end
 
     if $mount_users_dir
-      config.vm.synced_folder $home, $home, id: "core", :nfs => true, :mount_options => ['nolock,vers=3,udp']
+      config.vm.synced_folder $home, $home, type: "nfs"
```

Comment:
Is there any specific reason why we need to sync the whole home dir? Couldn't we do something like: `config.vm.synced_folder $gopath+"/src", "/hostgopath", type: "nfs"` and then get rid of `${hostGopath}` in `setup.sh`?